client/src/plugins/libparsec/definitions.ts

```diff
@@ -691,6 +699,12 @@ export interface LibParsecPlugin {
         recipient: UserID,
         role: RealmRole | null
     ): Promise<Result<null, ClientWorkspaceShareError>>
+    clientInfo(
+        client: number
+    ): Promise<Result<[OrganizationID, DeviceID, UserID, UserProfile, HumanHandle | null], ClientWorkspaceInfoError>>
+    clientDisplay(
```

Comment:
This function will not be used in the front, no need to add it.

Reply:
Done